src/python/foglamp/admin_api/controller.py

```diff
@@ -0,0 +1,15 @@
+from aiohttp import web
+from .model import User
+from .login import register_handlers, auth_middleware
+from .app_builder import build as build_app
+import asyncio
+
+
+def start():
+    """Create a http server for REST listening on port 8080"""
+    # TODO Read port from config (but might use nginx or gunicorn in production)
```

Comment:
nginx will be for static content only. here we will use aiohttp only (**may be** gunicorn)